bin/codemods/src/guided-tours-render.js

```diff
@@ -0,0 +1,55 @@
+/** @format */
+
+/**
+ * Guided Tours Rendering Codemod
+ * Transform directly inlined JSX markup into render props. That makes initial load
+ * faster and the Guided Tours get properly translated.
+ */
+
+const config = require( './config' );
+const prettier = require( 'prettier' );
+
+export default function transformer( file, api ) {
+	const j = api.jscodeshift;
+	const root = j( file.source );
+
+	/* Does the file have any <Step> JSX instances? */
+	const stepEls = root.findJSXElements( 'Step' );
+	if ( stepEls.size() === 0 ) {
+		// nothing to transform here
+		return null;
+	}
+
+	stepEls.forEach( stepEl => {
+		/* Extract the children */
+		const stepElValue = stepEl.get().value;
+		const { children } = stepElValue;
+
+		/* Wrap them in a functional component with Fragment */
+		const trIden = j.identifier( 'translate' );
+		const trProp = j.property( 'init', trIden, trIden );
+		trProp.shorthand = true;
+		const fragmentIden = j.jsxIdentifier( 'Fragment' );
```

Comment:
any reason you chose `Fragment` over `<>`?

Reply:
`<>` requires Babel 7.0 (currently in beta), the 6.x version doesn't understand it.

Also, my Atom editor reports this as a syntax error. I don't know where Atom's JavaScript parser is extendable by packages and how it works in general.

The only part of the toolset that's ready seems to be Prettier :smile: